### ISSUE TITLE
Many "Min vid" in Add-ons context menu

### ISSUE THREAD

**Issue by dfaerch** [meandavejustice/min-vid#891]
Right clicking on a page & choosing "Add-ons" shows this list, which seems rather useless. 

![image](https://user-images.githubusercontent.com/7899484/26922011-193ab34e-4c3e-11e7-8528-f06664a37289.png)

**Comment by meandavejustice** [comment]
@dfaerch I've only seen this when in between hot reloads when i'm developing min vid locally. Are you seeing this with a version you got from testpilot? what sites are you seeing this on?

**Comment by dfaerch** [comment]
@meandavejustice Yes, testpilot version. And it was on all sites, in all tabs. I had ~150 tabs open and some were idle youtubes. The menu-list persisted between shutting down and opening Firefox again (with the last session of course). But now it has gone and i really don't know what i did to get it to appear, so unfortunately its unreproducible.

**Comment by BoteRock** [comment]
this has just happened to me, even without youtube tabs open:

![image](https://user-images.githubusercontent.com/1028756/27092661-bafc3a10-5029-11e7-91ef-f25b6bda7d40.png)

don't know what it could be, but in both cases it seems to be 16 items.

**Comment by jean** [comment]
I'm seeing it here now too (since it broke `right-click a` as a quick way to select a link, by squatting on `a`). 
Also 16 items, in all browser windows. Min Vid 0.3.8

**Comment by jtojnar** [comment]
This is extremely annoying. Would it be possible add an option to disable context menus since I never use them?
